app/packages/looker/src/elements/util.ts

```diff
@@ -57,7 +52,8 @@ export function createElementsTree<
   update: StateUpdate<State>,
   dispatchEvent: (eventType: string, details?: any) => void
 ): Element {
-  const element = new root.node(config, update, dispatchEvent);
+  const element = new root.node();
+  element.boot(config, update, dispatchEvent);
```

Comment:
Whats the point of this change?

Reply:
There is discrepancy between how transpilation happens between Teams and OSS. There may be a setting that avoids this, but I don't know of it.

The problem is with classes and property initialization. Perhaps this was an antipattern to begin with, but a subclass that does not define its own constructor will have property initialization happen after parent construction.
```ts
abstract class A {

  constructor() {
    this.init();
  }

  abstract init(): void
}

class B extends A {
  private value: number;

  init() {
    this.value = 0;
  }
}
  
if (B().value !== 0) throw new Error("woe is me")
```